src/entities/calendarEvent/calendarEvent.inputs.ts

```diff
@@ -32,15 +34,15 @@ export class CreateCalendarEventInput {
 }
 
 @InputType()
-export class ListCalendarEventInput {
+export class ListCalendarEventsInput {
   @Field(() => ID, { nullable: true })
-  _id?: Types.ObjectId
+  calendar?: Types.ObjectId
 
   @Field(() => String, { nullable: true })
```

Comment:
@frankmangone shouldn't this be a graphql date / date time type?

Reply:
Oh, no, it's as it should be. Think of what you'd your query would be like. You'd want query, for example, the CalendarEvents belonging to a Calendar. Maybe we could make this more complex in the future, but for now, it should suffice, along with the ability to list CalendarEvents followed by Users